### ISSUE TITLE
Ensure Data Controller is clear

### ISSUE THREAD

**Issue by alaninmcr** [seek4science/seek#663]
At the moment it is not obvious who/what is the Data Controller for FAIRDOMHub. It should be obvious,.

See also https://github.com/workflowhub-eu/about/issues/16

**Comment by stuzart** [comment]
The fairdomhub conditions and policy was written by German lawyers to abide by German law, so I'd be very reluctant to change it at all.

**Comment by alaninmcr** [comment]
I agree - but it should be totally clear and it isn't to me (and probably others)

**Comment by stuzart** [comment]
it may be covered by https://fairdomhub.org/home/imprint , which is a legal requirement in Germany;  @muellerwHD will know better

**Comment by floradanna** [comment]
If by Data Controller you mean "who is responsible for the content that users upload in FAIRDOMHub", it should be the user himself/herself, as clarified in Terms and Conditions (T&C).
However, T&C say that if inappropriate content is detected, the provider can remove the content completely or partly.

If by Data Controller you mean "who is responsible for the personal data of registered/non-registered users" instead, it should be FAIRDOMHub legal entity (which is HITS gGmbH ?). https://fairdomhub.org/home/imprint
